Updating IMU comparison and schematics (#312)
SlimeVR/SlimeVR-Docs-Site: src/diy/imu-comparison.md

This page is based on user feedback and weighed against the survey, with scores showing the average answer given to the chip.
Following completing your own trackers, you can complete an extended version of that original survey yourself, or view previous responses on the [SlimeVR Experience Survey page](../misc/survey.md).

> ## Index (in order of best to worst)

Reviewer: ```suggestion
## Index (in order of best to worst with current firmware)
```

For clarification purposes: If 3 out of 10 chips are dead on arrival or die during early use, we refer to that as poor build quality.

## General Recommendations

> At the moment there is no general IMU recomended for all price ranges. although below there will be a short, general guide on what to get depending on your price range:

Reviewer: ```suggestion
At the moment there is no general IMU recomended for all price ranges. Although below there will be a short, general guide on what to get depending on your price range:
```

>     low range budget - BMI270 or LSM6DSO
> 
>     mid range budget - ICM-45686 - LSM6DSV
> 
>     high range budget - BNO085

Reviewer 2: ```suggestion
    Low range budget - BMI270 or LSM6DSO

    Mid range budget - ICM-45686 or LSM6DSV

    High range budget - BNO085
```

|Reset time |Cost |Availability|Build quality|
|:---------:|:---:|:----------:|:-----------:|

> |45-60 min|~$10 |Sufficient  |Excellent    |

Reviewer: ```suggestion
|30-45 min|~$11 |Sufficient  |Excellent    |
```

Idk where you got BNO085 for $10. The cheapest is $11 on our site. Also the drift time is very subjective and there are a lot of arguments, so I wouldn't recommend increasing it for BNO just yet. We really need a well-defined measurement system for drift time...

Score: <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star-half-o"></i>

|Pros          |Cons                                   |
|--------------|---------------------------------------|
|Accurate      |Expensive                              |
|Reliable      |Big                                    |
|Smooth        |                                       |

> Please note, if you are looking for BNO085s SlimeVR may have [spares for sale that can be found here](https://shop.slimevr.dev/products/slimevr-imu-module-bno085).

---
## LSM6DSV
The LSM6DSV is a very good IMU for DIY SlimeVR.

> it seems to perform just as good as the BNO for a lower cost!

Reviewer 2: ```suggestion
It seems to perform just as well as the BNO for a lower cost.
```

|Reset time |Cost |Availability|Build quality|
|:---------:|:---:|:----------:|:-----------:|
|45-60 min  |~$8  |Insufficient|Great        |

Score: <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star-half-o"></i>

|Pros          |Cons                                   |
|--------------|---------------------------------------|
|Accurate      |Expensive                              |
|Reliable      |Experimental                           |
|Smooth        |                                       |

---
## ICM-45686
The ICM-45686 is a potentially very good and <b>very experimental</b> IMU for DIY SlimeVR.

> So far it is shown to be testing on around the same level as the BNO/DSV but has only been tested by a few people so results are not entirely conclusive yet

Reviewer 2: ```suggestion
So far it is shown to be testing on around the same level as the BNO/DSV but has only been tested by a few people so results are not entirely conclusive yet.
```

|Reset time |Cost |Availability|Build quality|
|:---------:|:---:|:----------:|:-----------:|
|45-60 min|~$6 |Sufficient  |Great           |

Score: <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star-half-o"></i>

|Pros          |Cons                                                                     |
|--------------|-------------------------------------------------------------------------|
|Cheap         |Very Experimental                                                        |
|Reliable      |Only available in module form from a single source                       |
|Smooth        |                                                                         |

---
## LSM6DSO
The LSM6DSO is a good IMU for DIY SlimeVR.

> it has been shown to be slightly better than the BMI270 at a slightly higher cost.

Reviewer 2: ```suggestion
It has been shown to be slightly better than the BMI270 at a slightly higher cost.
```

|Reset time |Cost |Availability|Build quality|
|:---------:|:---:|:----------:|:-----------:|
|25-35 min|~$3.5 |Sufficient  |Great         |

Score: <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star-o" ></i>

|Pros          |Cons                                                                     |
|--------------|-------------------------------------------------------------------------|
|Cheap         |Experimental                                                             |
|Reliable      |Only available in module form from a single source                       |

> |Smooth        |Poor Sensitivity                                                         |

Reviewer 2: ```suggestion
|Smooth        |Poor sensitivity                                                         |
```

---
## BMI270
The BMI270 is a good IMU for DIY SlimeVR.
It seems to perform significantly better than the BMI160 while still being affordable.

|Reset time |Cost  |Availability|Build quality|
|:---------:|:----:|:----------:|:-----------:|
|20 - 30min |~$2.5 |Sufficient  |Great        |

Score: <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star-o" ></i>

|Pros                   |Cons                                                            |
|-----------------------|----------------------------------------------------------------|
|Cheap                  | Only available in module form from a single source             |

> |Reliable               | Experimental                                                   | 

Reviewer 2: I don't know if BMI270 should still be considered experimental, after an entire year of being available.

|Smooth                 |                                                                |
|Single time calibration|                                                                |

---
## BMI160

<b>Not recommended for new designs.</b>

The BMI160 is currently not recomended for new slime trackers.

> it is very outdated with alot of newer IMUs surpassing it in price to performance

Reviewer 2: ```suggestion
It is very outdated with a lot of newer IMUs surpassing it in price to performance.
```
